versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractReferences.java

```diff
@@ -246,6 +246,9 @@ void assign() throws Exception {
 
   @Test
   void recreateDefaultBranch() throws Exception {
+    // note: the default branch can not be deleted through the TreeApi,
+    // but the underlying DatabaseAdapter should still support this operati
```

Comment:
typo: `operati` (maybe not everything was copied?)

also maybe we should think about this more and also add WHY the db adapter needs to be able to still do this when its not possible form the user facing apis :thinking:

Reply:
Good point, I think that the main reasoning is that the underlying db adapter is a low-level API that may allow some internal code to remove the branch. On the other hand, we don't want to allow the same for the user-facing APIs.